src/components/DeleteEditButtons/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import { Button } from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
+import EditIcon from '@material-ui/icons/Edit';
+import { withStyles } from '@material-ui/core/styles';
+import styles from './styles';
+
+import * as ROUTES from '../../constants/routes';
+import { deleteEventById } from '../../services/events';
+
+function DeleteEditButtons(props) {
```

Comment:
Add todo for confirmation modal for delete: we don't want people accidentally deleting events :)
Later on it might be helpful to make a ButtonWithConfirmationModal component